web-elements/src/main/java/page/TextBoxPage.java

```diff
@@ -0,0 +1,59 @@
+package page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+public class TextBoxPage extends AbstractPage {
+    private static final String TEXT_BOX_PAGE_URL = "https://demoqa.com/text-box";
```

Comment:
You can reuse base url from parent, as example
TEXT_BOX_PAGE_URL = baseurl + "/text-box"